config/schedule.rb

```diff
@@ -7,10 +7,15 @@
 set :output, "/tmp/figgy_update_bib_ids.log"
 set :job_template, "bash -l -c 'export PATH=\"/usr/local/bin/:$PATH\" && :job'"
 job_type :logging_rake, "cd :path && :environment_variable=:environment bundle exec rake :task :output"
+
 every :day, at: "11:00 PM", roles: [:db] do
   logging_rake "figgy:update_bib_ids"
 end
 
+every :monday, at: '10am', roles: [:db] do
+  rake "figgy:send_collection_reports"
```

Comment:
`logging_rake` ensures that `$PATH` is exported properly for the `rake` invocation

Reply:
I didn't seem to need it; the job ran okay from cron!